readme.md

```diff
@@ -53,7 +53,51 @@ Set maximum image quality factor. (`0`-`100`).
 
 Type: `number` `string`
 
-Try to optimize file to given size. Target size is specified either in kilo bytes (`1`-) or as percentage (`1%`-`99%`).
+Try to optimize file to given size. Target size is specified either in kilo bytes (`1`-`n`) or as percentage (`1%`-`99%`).
+
+##### stripAll
+
+Type: `boolean`  
+Default: `true`
+
+Strip all markers from output file. If you want to control what markers are stripped, this _must_ be set to `false`.
+
+##### stripCom
+
+Type: `boolean`  
+Default: `true`
+
+Strip Comment markers from output file. Requires `stripAll` to be `false`.
```

Comment:
Hmm, I think the note under the `stripAll` option is sufficient. Remove the `Requires stripAll to be false.` from these ones.